administration/src/errors/DefaultErrorMap.tsx

```diff
@@ -23,10 +23,16 @@ type ErrorExtensions = {
 const defaultErrorMap = (extensions?: ErrorExtensions): GraphQLErrorMessage => {
   const defaultError = { title: 'Etwas ist schief gelaufen.' }
 
-  if (!extensions || !extensions['code']) {
+  if (!extensions || extensions.code === undefined) {
     return defaultError
   }
-  switch (extensions['code']) {
+
+  const cardInfo = extensions.encodedCardInfoBase64
+    ? CardInfo.fromBinary(base64ToUint8Array(extensions.encodedCardInfoBase64))
+    : null
+  const codeTypeText = extensions.codeType === CodeType.Dynamic ? 'Aktivierungscode' : 'statische QR-Code'
```

Comment:
Wouldn't it be better to keep it in the only case that it is used? Maybe you need to introduce a variable scope as in
```ts
  case GraphQlExceptionCode.InvalidQrCodeSize: {
      const cardInfo = CardInfo.fromBinary(base64ToUint8Array(extensions.encodedCardInfoBase64!))
      const codeTypeText = extensions.codeType === CodeType.Dynamic ? 'Aktivierungscode' : 'statische QR-Code'
      ...
     }
```

Reply:
TIL. And also that I did not read the docs of the rule properly. Thanks!